uber/models.py

```diff
@@ -1805,12 +1805,18 @@ def track_pageview(cls, url, query):
             if "registration" in url:
                 with Session() as session:
                     attendee = session.query(Attendee).filter(Attendee.id == params['id']).first()
-                    Tracking.track(c.PAGE_VIEWED, attendee)
+                    if attendee is None:
+                        return
+                    else:
+                        Tracking.track(c.PAGE_VIEWED, attendee)
             # Looking at a group's details
             elif "groups" in url:
                 with Session() as session:
                     group = session.query(Group).filter(Group.id == params['id']).first()
-                    Tracking.track(c.PAGE_VIEWED, group)
+                    if group is None:
```

Comment:
Same here as with my `if not attendee` comment above - `if not group` is more Pythonic, I believe.